riva/clients/tts/riva_tts_client.cc

```diff
@@ -37,6 +39,7 @@ DEFINE_string(ssl_client_key, "", "Path to SSL client certificates key");
 DEFINE_string(ssl_client_cert, "", "Path to SSL client certificates file");
 DEFINE_int32(rate, 44100, "Sample rate for the TTS output");
 DEFINE_bool(online, false, "Whether synthesis should be online or batch");
+DEFINE_bool(streaming, false, "Whether synthesis should be streaming or batch");
```

Comment:
is it possible to use existing `online` flag for this? separate flags for input/output can be confusing for user. Do we see a case where we want to support non-streaming input and streaming output?